### PR TITLE
Simpspector analyser

### LINKED ISSUE #44: split SimpSpector in two projects
I think it is nice to split SimpSpector in two projects. In an analysis tool like `SimpSpector-CLI` and a web interface. The advantage is that the cli tool has all the dependencies to the other tools and you can write easier tests.

### LINKED ISSUE #108: Switching filename to .simpspector.yml (like travis, scruitinizer and other)
from tolry:

"Not specific to this PR, but: what about switching the filename to ".simpspector.yml" as travis, scrutinizer and others do? Keeps the "normal" file listing clean and denotes that this file might not directly important for users. Any thoughts?"

### PR DESCRIPTION
**Pull request by DavidBadura**
btw change simpspector config file from 'simpspector.yml' into '.simpspector.yml'.

fixes #44 
fixes #108

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,10 +21,8 @@
...
         "zeichen32/gitlabapibundle": "~2.0",
         "igorw/get-in": "~1.0",
-        "phpmd/phpmd": "~2.1",
-        "squizlabs/php_codesniffer": "~2.0",
         "timetoogo/pinq": "~3.0",
-        "nikic/php-parser": "~1.0.2",
+        "simpspector/analyser": "dev-master",
         "davidbadura/markdown-builder": "~1.0",
         "gremo/ciconia-bundle": "dev-twig-extension",
         "asm89/twig-lint": "dev-fix/twig-gt-1.16.2"
```
Comment (tolry) [inline composer.json]: can be removed from webinterface

File: src/SimpleThings/AppBundle/CommitHandler.php [hunk 2]
```diff
@@ -83,7 +86,7 @@ public function handle(Commit $commit)
 
             $workspace = $this->gitCheckout->create($commit, $logger);
 
-            $workspace->config = $this->configLoader->load($workspace);
+            $workspace->config = $this->configLoader->load($workspace->path . '/simpspector.yml');
```
Comment (tolry) [inline src/SimpleThings/AppBundle/CommitHandler.php]: Not specific to this PR, but: what about switching the filename to ".simpspector.yml" as travis, scrutinizer and others do? Keeps the "normal" file listing clean and denotes that this file might not directly important for users. Any thoughts?

**Comment by tolry** [comment]
Looks good to me. Very little changes to the original execution logic were needed, I like it.

File: src/SimpleThings/AppBundle/CommitHandler.php (hunk 2 above)
Comment (DavidBadura) [inline src/SimpleThings/AppBundle/CommitHandler.php]: :+1:

File: composer.json (hunk 1 above)
Comment (tolry) [inline composer.json]: ok cannot be removed before  twig-lint PR is merged into main repo

**Comment by tolry** [comment]
still missing: event system will be moved to analyser

**Comment by smoench** [comment]
MarkdownBuilderTest should be (re)moved

**Comment by smoench** [comment]
@DavidBadura ping

**Comment by DavidBadura** [comment]
so, @simpspector/owners this pull request is ready and big enough.
the metrics are stored but not used yet. this should be done in another pull request.

**Comment by tolry** [comment]
:+1: 

no time to review - we can refactor everything on the hackday anyway :smile:

**Pull request merged**